packages/vite/src/node/env.ts

```diff
@@ -69,6 +78,8 @@ export function loadEnv(
     }
   }
 
+  debug?.(`using resolved env: %O`, env)
```

Comment:
Should we show `process.env` values (internal values) too? What do you think?

Reply:
I think this current log is fine for now 👍